lldb/include/lldb/Core/Telemetry.h

```diff
@@ -29,6 +30,9 @@ namespace telemetry {
 
 struct LLDBEntryKind : public ::llvm::telemetry::EntryKind {
   static const llvm::telemetry::KindType BaseInfo = 0b11000;
+  static const llvm::telemetry::KindType DebuggerInfo = 0b11001;
```

Comment:
These values are kind of random. Any reason to not just one `1`, `2`, `3`, etc?

Reply:
Yes, these have common base so we can do a quick mask on all the subclasses's kind.
(See LLDBBaseTelemtryInfo::classof(), line 55):

```
 static bool classof(const llvm::telemetry::TelemetryInfo *t) {
    // Subclasses of this is also acceptable.
    return (t->getKind() & LLDBEntryKind::BaseInfo) == LLDBEntryKind::BaseInfo;
  }
```